.circleci/config.yml

```diff
@@ -0,0 +1,52 @@
+version: 2
+
+jobs:
+  build_and_test:
+
+    machine:
```

Comment:
@duozersk can you explain what is being done here and why we use this image?

Reply:
We need to use `machine` executor to run docker-compose based builds.
See https://circleci.com/docs/2.0/docker-compose/#using-docker-compose-with-machine-executor and https://circleci.com/docs/2.0/executor-types/#using-machine